### PR TITLE
Add set of new icons

### PR DESCRIPTION
**Pull request by matzimowski**
screen shot 2016-08-30 at 15 58 21

### CODE REVIEW

**Comment by aju** [comment]
Ok

**Comment by kdzwinel** [comment]
OK

File: src/sass/_config.scss [hunk 1]
```diff
@@ -1,5 +1,4 @@
-/*
- DO NOT USE THESE COLORS DIRECTLY!
+/* DO NOT USE THESE COLORS DIRECTLY!
```
Comment (vergilius) [inline src/sass/_config.scss]: this might be removed, as we do not follow this rule anymore

**Comment by vergilius** [comment]
👍

File: src/sass/_config.scss (hunk 1 above)
Comment (matzimowski) [inline src/sass/_config.scss]: clever one!

**Comment by aju** [comment]
Ok

**Pull request merged**